public/css/currency-symbol-style.css

```diff
@@ -0,0 +1,6 @@
+/* currency-symbol-style.css
+/* This style is applied on currency symbol */
+.currency-symbol-style {
+    color: green;
+    /* border: solid red 1px; */
+    } 
```

Comment:
Why do we need this?

Reply:
We provided this for styling on the currency symbol.

Reply:
Remove this also, we are not giving any styling because this will force us to change the directive to v-html.

Reply:
Styling has been removed, and the files where 'v-html' was used have been restored to their previous state.